### PR TITLE
Add RaspberryPI Zero pin diagram library

### PR DESCRIPTION
**Pull request by marcottebear**
I add a library for RaspberryPI Zero pins based on the existing RaspberryPi3 by @revolunet :)

### CODE REVIEW

**Comment by revolunet** [comment]
Preview image is very heavy

**Comment by lipis** [comment]
Could you rotate the image and have a maximum dimension of 400px for the preview?

File: libraries.json [hunk 1]
```diff
@@ -178,5 +178,14 @@
...
     "preview": "drwnio/drwnio.png",
     "date": "2021-01-25"
+  },
+  {
+    "name": "RaspberryPI Zero",
+    "description": "A RaspberryPI Zero pin diagram",
+    "author": "marcottebear",
+    "url": "https://github.com/marcottebear",
+    "source": "marcottebear/raspberrypi_zero.excalidrawlib",
+    "preview": "marcottebear/raspberrypi_zero.png",
```
Comment (lipis) [inline libraries.json:188]:
```suggestion
    "source": "marcottebear/raspberrypi-zero.excalidrawlib",
    "preview": "marcottebear/raspberrypi-zero.png",
```

Comment (lipis) [reply libraries.json:188]: Could you also rename the files to use `kebab-case`?

**Comment by marcottebear** [comment]
I would like to add also (in another PR) some other drawings related to Raspberry like ADS1115 board and other stuff.. maybe we can create a unique directory to store them? If you like we can combine our dirs for RaspberrPI @revolunet , what do you think?

**Comment by revolunet** [comment]
Yeah, i guess it would be more user-friendly to have a lib containing all related drawings

**Comment by lipis** [comment]
We are open for suggestions.. we just want to make sure to give credit where credit's due. So one idea is to create a raspberrypi directory and have all elements in one library, and there we can add a `README.md` with the contributor's names and link to that.. WDYT?

**Comment by marcottebear** [comment]
Think in the long term divide the libraries based on the Author can lead to a bit of a mess maybe... Of course, mentions the author is important too! Think they could be organized by topic and for each include a list of authors... in the `README.md` or maybe also reorganizing `libraries.json` (but is a bigger change 😅 ), don't know which is the best really... 

I just suggested creating a common directory with the authors' names (I thought this was the 'convention') in order to just put all the Raspberry related items in one place and easily search on `https://libraries.excalidraw.com/`, I see there are some items with two authors...

**Comment by revolunet** [comment]
mention in readme is very fine for me

**Comment by dwelle** [comment]
Was there mentioned simply listing both (or more) authors as creators of a given library?

image


Also note that long-term we plan to make library items be searchable and installable individually. This will improve the UX in case we add all the rpi items into a single library, but a user will want to install just one instead of importing all of them at once.

**Comment by lipis** [comment]
The current model doesn't scale for multiple authors..

**Comment by marcottebear** [comment]
If we fill the "authors" field in the `libraries.json` will not enough? (I didn't look how is rendered really... I'm guessing...)
Like [this](https://github.com/excalidraw/excalidraw-libraries/blob/06814206768521e31c8916d9b95b62c5bd554ee3/libraries.json#L32)...

Was just a little improvement to have everything in the same place, if it's not possible I think is fine anyway :)

**Comment by dwelle** [comment]
I suppose it's because we namespace each library under author's username folder. Ultimately it's blocked by how we load libraries (single URL): https://github.com/excalidraw/excalidraw-libraries/blob/2d0cf1709d927d02ffccfda2de62dc976d5ad80b/script.js#L83

We could either precompile the library files and commit as joined library file from two (or more) authors, or we could support loading libraries from multiple urls at once.

Or, as a temp workaround we can simple create a folder that's named: `author1_autor2/`.

Either way I think long-term we'll definitely want to support cases where multiple authors contributed to a single library.

**Comment by lipis** [comment]
we can create a directory `raspberypi` and have a `README.md` there and all the related libs as separate files.. why not? To keep it simple..

**Comment by dwelle** [comment]
I wouldn't say creating a subfolder with its own readme and everything is keeping it simple :).

**Comment by lipis** [comment]
> I wouldn't say creating a subfolder with its own readme and everything is keeping it simple :).

This will be just a link from the author could be: `@Raspberry Pi Team` with a link to the readme for the contributors..

Adding/Supporting multiple author won't scale with the current (directory based) model..

I'm just wondering is it OK to include all the Raspberry Pi boards in one library or we should have multiple.. if having multiple, we can continue as we are and just merge this PR as is.

**Comment by lipis** [comment]
Closing in favor of #44 let's continue there.

**Pull request closed without merging**